logic/src/commonMain/kotlin/com/wire/kalium/logic/data/event/Event.kt

```diff
@@ -256,15 +256,20 @@ sealed class Event(open val id: String) {
 
         data class Update(
             override val id: String,
-            val userId: String,
-            val accentId: Int,
-            val ssoIdDeleted: Boolean
+            val userId: String?,
+            val accentId: Int?,
+            val ssoIdDeleted: Boolean?,
+            val name: String?,
+            val handle: String?,
+            val email: String?,
         ) : User(id) {
             override fun toString(): String {
                 return "id: ${id.obfuscateId()} " +
                         "userId: $userId " +
                         "accentId: $accentId " +
-                        "ssoIdDeleted: $ssoIdDeleted"
+                        "name: $name" +
```

Comment:
We don't want to print the `ssoIdDeleted` field?

Reply:
I don't know 🕳️  so I just removed everything not necessary, I don't think brings value more than for developing now.